.github/ISSUE_TEMPLATE/compatibility_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Compatibility request
+about: Suggest supporting a new platform version OpenSearch
+title: 'Compatibility with OpenSearch (version)'
+labels: compatibility
+assignees: ''
+
+---
+
+## Description
+We need to ensure the UI compatibility with the next version of OpenSearch vX.X.
+This update is still being discussed, but we need to be aware of potential issues.
+
+For that, we need to:
+
+- [ ] Review opensearch and opensearch-dashboard latest stable changelog. 
+- [ ] Identify improvements and potential impact on the UI.
+- [ ] Update version with changes made in Wazuh
```

Comment:
```suggestion
- [ ] Create new tracking and development branches.
```

Reply:
Check the Wazuh dashboard fork documentation to learn more about tracking and development branches.